fork_test

```diff
@@ -0,0 +1 @@
+fork test
```

Comment:
Can you add more explanations about the functionality of this part?

Reply:
Okay.